defaults/krill.conf

```diff
@@ -249,4 +249,36 @@
 # timing_child_certificate_valid_weeks = 52
 # timing_child_certificate_reissue_weeks_before = 4
 # timing_roa_valid_weeks = 52
-# timing_roa_reissue_weeks_before = 4
\ No newline at end of file
+# timing_roa_reissue_weeks_before = 4
+
+#
+#                 Suspend inactive child CAs
+#
+# NOTE: YOU DO NOT NEED THIS UNLESS YOU DELEGATE RESOURCES TO CHILD CAS
+#
+# If a child CA is deactivated, then it will no longer update its publication
```

Comment:
Deactivated by whom? The operator of this Krill instance or the operator of the child CA? I think the latter. Perhaps just instead say if the child CA no longer updates its publication point?